File: Array and String/Larget_Number_Twice.cpp
class Solution {
public:
    int dominantIndex(vector<int>& nums) {
        int max1 = 0;
        int index = 0;
        for(int i = 0; i<nums.size(); i++)
        {
            if(max1<nums[i])
            {
                max1 = nums[i];
                index = i;
            }
        }
        int flag = 0;
        for(int i = 0; i<nums.size(); i++)
            if(2*nums[i] <= max1)
                flag++;
        if(flag == nums.size()-1)
            return index;
        else
            return -1;
    }
};

File: Larget_Number_Twice.py
class Solution:
    def dominantIndex(self, nums: List[int]) -> int:
        max1 = 0
        index = 0
        for i in range(len(nums)):
            if max1 < nums[i]:
                max1 = nums[i]
                index = i
        flag = 0
        for i in nums:
            if 2*i <= max1:
                flag+=1
        if flag == len(nums)-1:
            return index
        else:
            return -1
                